src/ui/PrimitiveList.tsx

```diff
@@ -24,11 +33,13 @@ class Primitive extends Component {
     connectDragSource: PropTypes.func.isRequired,
   }
 
-  handleKeyDown = e => {
+  handleKeyDown = (e: React.KeyboardEvent) => {
     switch (SHARED.keyMap[SHARED.keyName(e)]) {
     case 'copy':
       e.preventDefault();
-      copyNodes([this.props.primitive]);
+      // TODO(pcardune): copy() only works with ASTNodes,
+      // so I don't know how this ever worked before...
+      copy([this.props.primitive as any]);
```

Comment:
This is why I'm not sure using `copy` instead of the non-existent `copyNodes` is correct. `copy()` appears to only support copying `ASTNode` objects and not `Primitive` objects.